media/js/helpfulness.js

```diff
@@ -0,0 +1,91 @@
+(function(win, doc, $) {
+
+    // this feature requires localStorage
+    if (('localStorage' in win)) {
+        var ignore = localStorage.getItem('helpful-ignore') === 'true'; // true if ever clicked ignore
+        var asked_recently = parseInt(localStorage.getItem(doc.location + '#answered-helpful'), 10) > Date.now();
+        if (!ignore && !asked_recently) {
+            // ask about helpfulness after 1 min
+            setTimeout(inquire, 60000);
+        }
+    }
+
+    // create a notification
+    function inquire() {
+        // dimension7 is "helpfulness"
+        ga('set', 'dimension7', 'Yes');
```

Comment:
My :dollar: 0.02:

The point of the custom dimension is to analyze between 2 segments/cohorts in GA: 1. Those who saw the helpfulness feature and 2. Those who did not.

Given that, only setting the dimension when we create the (initial) notification makes sense to me. Otherwise, we'll be putting visitors into the "Saw helpfulness dialog" cohort even if they never saw the notification.